src/patterns/UserManagement/UserManagement.tsx

```diff
@@ -56,19 +57,29 @@ const UserManagement = forwardRef<HTMLDivElement, UserManagementProps>(
 
     const isLoggedIn = authState === AuthState.LoggedIn;
     const shouldShowAccountDetails = authState !== AuthState.Loading;
+    const ResponsiveAccountIcon = () => (
```

Comment:
After seeing what this looks like, maybe a simpler fix would be to use scss with media queries?  @aahill50  After thinking about it, I think SSRMediaQuery is best used when the DOM elements will actually be different

Reply:
Alright, I changed it to using css